sdk/trace/src/main/scala/org/typelevel/otel4s/sdk/trace/data/EventData.scala

```diff
@@ -104,6 +107,7 @@ object EventData {
       timestamp: FiniteDuration,
       exception: Throwable,
       attributes: Attributes,
+      limitAttributes: Attributes => LimitedData[Attribute[_], Attributes],
```

Comment:
What if we pass only `attributes: LimitedData[Attribute[_], Attributes]` here?
```suggestion
      attributes: LimitedData[Attribute[_], Attributes],
```

We will have a limited collection so there is no need to require the `limitAttributes` param:
```scala
Impl(ExceptionEventName, timestamp, attributes.prependAll(exceptionAttributes))
```

Reply:
Yes, you are right. Additional `prependAll` method does the trick. Adjusted.